src/main/kotlin/no/nav/familie/ef/iverksett/arbeidsoppfolging/SendVedtakTilArbeidsoppfølgingTask.kt

```diff
@@ -0,0 +1,38 @@
+package no.nav.familie.ef.iverksett.arbeidsoppfolging
+
+import no.nav.familie.ef.iverksett.infrastruktur.task.opprettNestePubliseringTask
+import no.nav.familie.ef.iverksett.iverksetting.IverksettingRepository
+import no.nav.familie.ef.iverksett.repository.findByIdOrThrow
+import no.nav.familie.prosessering.AsyncTaskStep
+import no.nav.familie.prosessering.TaskStepBeskrivelse
+import no.nav.familie.prosessering.domene.Task
+import no.nav.familie.prosessering.internal.TaskService
+import org.springframework.stereotype.Service
+import java.util.*
+
+@Service
+@TaskStepBeskrivelse(
+    taskStepType = SendVedtakTilArbeidsoppfølgingTask.TYPE,
+    beskrivelse = "Sender vedtaksperioder til arbeidsoppfølging.",
+    settTilManuellOppfølgning = true,
+)
+class SendVedtakTilArbeidsoppfølgingTask(
+    private val taskService: TaskService,
+    private val iverksettingRepository: IverksettingRepository,
+    private val arbeidsoppfølgingService: ArbeidsoppfølgingService,
+) : AsyncTaskStep {
+
+    override fun doTask(task: Task) {
+        val behandlingId = UUID.fromString(task.payload)
+        val iverksett = iverksettingRepository.findByIdOrThrow(behandlingId).data
+        arbeidsoppfølgingService.sendTilKafka(iverksett)
```

Comment:
Burde vi featuretoggle dette for å kunne deploye og sen ta i bruk i produksjon når vi har behov? 
Samtidig har vi ikke veldig mye som skjer på iverksett så går ju fint å bare deploye branchen i dev

Reply:
Var inne på tanken, men tror de vil være relativt raske på implementering og testing i dev dersom det er ute, så jeg tenkte kanskje å gå for alternativ 2. Bare deploye til dev også merge så fort de har fått lest melding i dev, og er fornøyde. Kontrakten er i utgangspunktet avklart, så det vil antakelig gå helt fint å deploye til prod uten featuretoggle med en gang også.